pkg/didcomm/protocol/didexchange/persistence.go

```diff
@@ -67,14 +67,14 @@ func (c *ConnectionRecorder) SaveInvitation(verKey string, invitation *Invitatio
 //
 // Args:
 //
-// verKey: recipient key of the invitation object to be saved in store
+// id: invitation id
 //
 // Returns:
 //
 // invitation found
 // error: error
-func (c *ConnectionRecorder) GetInvitation(verKey string) (*Invitation, error) {
-	k, err := invitationKey(verKey)
+func (c *ConnectionRecorder) GetInvitation(id string) (*Invitation, error) {
+	k, err := invitationKey(id)
```

Comment:
You may have to remove computeHash() logic from inside invitationKey() function. We used computeHash since there was a possibility of having very long verKey. @sandrask

Reply:
I was thinking about it yesterday but decided not to do it for two reasons: 
1) UUIDs are long
2) we would be exposing database ids
We can further discuss and create an issue if necessary.